tests/e2e-instrumentation/instrumentation-apache-httpd/chainsaw-test.yaml

```diff
@@ -4,14 +4,25 @@ kind: Test
 metadata:
   name: instrumentation-apache-httpd
 spec:
-  namespaceTemplate:
-    metadata:
-      annotations:
-        openshift.io/sa.scc.uid-range: 1000/1000
-        openshift.io/sa.scc.supplemental-groups: 3000/1000
   steps:
   - name: step-00
     try:
+    - command:
+        entrypoint: kubectl
+        args:
+        - annotate
+        - namespace
+        - ${NAMESPACE}
+        - openshift.io/sa.scc.uid-range=1000/1000
+        - --overwrite
+    - command:
+        entrypoint: kubectl
+        args:
+        - annotate
+        - namespace
+        - ${NAMESPACE}
+        - openshift.io/sa.scc.supplemental-groups=3000/1000
+        - --overwrite
```

Comment:
Why are we changing this? Does the current form not work?

Reply:
So when a new project or namespace in created in OpenShift, the openshift.io/sa.scc.mcs automatically added with other scc annotations. If we set only the uid and group scc annotation via a resource file template, the openshift.io/sa.scc.mcs is not added with other required annotations and we get error in the deployments. 

`chainsaw-becoming-lab": unable to find annotation openshift.io/sa.scc.mcs`

So instead we create the namespace with the annotations auto added and only modify the uid and group scc ones which are required.

Reply:
We should add comments explaining this, then. It's far from obvious, and operator CI doesn't check for OpenShift breakages.

Reply:
@swiatekm-sumo I added comment to explain why the change is needed.